app/views/entries/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<html>
+<head>
+	<title>Add New Entry</title>
+	<link href="https://fonts.googleapis.com/css?family=Oswald|Noto+Sans+TC&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="new_entry_form.css">
+	<script type="text/javascript"></script>
+</head>
+<body>
+	<div class="mainPanel">
+		<h1>CHANG'S SLANG</h1>
+		<h2>New Entry</h2>
+		<%= form_for(@entry) do |f| %>
+			<ul>
+			<% @entry.errors.full_messages.each do |error| %>
```

Comment:
How would this have errors? 

EDIT: oh this is some rails thing? This is too much rails magic for me :(